product_docs/docs/pem/9/tuning_performance/performance_diagnostic.mdx

```diff
@@ -77,35 +77,48 @@ By default, the top Performance Diagnostic graph pulls the data of the last hour
+  Select a time on the Line graph to analyze the wait events. To show or hide a wait event type in all the graphs, select the graph legends. The analysis is simpler when you can only see the wait event types you want to analyze. 
 
--   The **SQL** tab displays the list of SQL queries having wait events for the selected sample time.
--   The **Users** tab displays the details of the wait events grouped by users for the selected sample time.
--   The **Waits** tab displays the number of wait events belonging to each wait event type for the selected sample time.
+  
+- The third section displays the wait event details in the Performance Diagnostics dashboard based on your selected duration in the second graph. It displays the wait event details on two tabs:
 
-To show or hide a wait event type in all the graphs, select the graph legends. The analysis is simpler when you can only see the wait event types you want to analyze. 
+  - The **Summary** tab displays the list of top 50 SQL queries with query id's, total CPU events, percentage of CPU usage, wait events, and execution count for the selected time interval. This SQL queries are grouped into four different categories:
+    
+    - Top queries by wait events &mdash; Lists the queries across all the sessions for the selected sample time that have spent the longest time in wait states. It displays the query ID, query, total wait events, waiting time in percentage, wait events and the execution count along with number of users executed that query in the number of databases.
+    
+    - Top queries by CPU usage &mdash; Lists the queries across all the sessions for the selected sample time that have used the most CPU time. It displays the query ID, query, total CPU events, CPU usage percentage, wait events, and execution count along with number of users executed that query in the number of databases. The data in the table is in decreasing order of the number of CPU events.
+    
+    - Top users by wait events &mdash; Lists the users across all the sessions for the selected sample time whose queries have spent the longest time in wait states. It displays the usernames, database name, total wait events, waiting time percentage, wait events and the number of queries that have spent the longest time in wait states.
+    
+    - Top users by CPU usage &mdash; Lists the users across all the sessions for the selected sample time whose queries have used the most CPU time. It displays the username, database name, total CPU events, CPU usage percentage, wait events, and the number of queries that have used the most CPU time.
 
-You can filter the data displayed in the rows under all three tabs. You can also sort the data alphabetically by selecting the column headers.
+  - The **Sample Selection** tab displays the details of the wait events for the selected sample time grouped by:
```

Comment:
This information is missing.

`
Select a time on the Line graph to analyze the wait events. The third section displays the wait event details in the Performance Diagnostics dashboard based on your selected duration in the second graph.
`